### ISSUE TITLE
"Internal error for --format" popup in latest EAP and shellcheck

### ISSUE THREAD

**Issue by singalen** [pwielgolaski/shellcheck-plugin#7]
I got a popup: "Internal error for --format. Please file a bug :("
I'm using the latest CLion EAP (#CL-191.6014.11, built on March 6, 2019) on macOS 10.14.2, and a latest `shellcheck` that I built yesterday from source git/master.

In `idea.log`, I only see one relevant line:
```
2019-03-06 10:16:53,755 [ 115872]   INFO - llcheck.utils.ShellcheckRunner - Running command: /Users/me/bin/shellcheck --format=json /Users/me/bin/utils/merger 
```

**Comment by pwielgolaski** [comment]
Isn’t it shellcheck issue?

**Comment by singalen** [comment]
Sorry, you're right. Will report upstream.
It does produce the error, even if after it emits a valid JSON with diagnostics.
